pkg/machinepool/helper.go

```diff
@@ -390,3 +405,54 @@ func (r *ReplicaSizeValidation) MaxReplicaValidatorOnClusterCreate() interactive
 		)
 	}
 }
+
+func GetZoneType(machinePool *cmv1.MachinePool) string {
+	zones := machinePool.AvailabilityZones()
+	if len(zones) == 0 {
+		return ZoneTypeNA
+	}
+
+	for _, zone := range zones {
+		zoneLower := strings.ToLower(zone)
+
+		if strings.Contains(zoneLower, "wl") {
+			return ZoneTypeWavelength
+		}
+		if strings.Contains(zoneLower, "outpost") {
+			return ZoneTypeOutpost
+		}
+		if strings.Contains(zoneLower, "-lz") || strings.Count(zoneLower, "-") > 3 {
+			return ZoneTypeLocalZone
+		}
+	}
+
+	return ZoneTypeStandard
+}
+
+func IsWinLIEnabled(labels map[string]string) string {
```

Comment:
These funcs are only used in the `machinepool` package, by the looks of it we could make these private to help reduce clutter in the `machine pool` pkg